crates/tabby/Cargo.toml

```diff
@@ -51,4 +51,4 @@ link_shared = ["ctranslate2-bindings/link_shared"]
 scheduler = ["tabby-scheduler"]
 
 [build-dependencies]
-vergen = { version = "8.0.0", features = ["build", "git", "gitcl"] }
+vergen = { version = "8.0.0", features = ["build", "git", "git2"] }
```

Comment:
is this required? could you link the related doc here?

Reply:
Doc is as added in PR description:
https://docs.rs/vergen/latest/vergen/#configuring-the-git-feature

Not making this change, errors will be encountered while compiling:
```
...
#16 10.52   Downloaded rust-stemmers v1.2.0
#16 10.59   Downloaded encoding_rs v0.8.32
#16 10.73   Downloaded utoipa-swagger-ui v3.1.3
#16 10.84 warning: no suitable 'git' command found!
#16 10.84 warning: VERGEN_GIT_BRANCH set to default
#16 10.84 warning: VERGEN_GIT_COMMIT_AUTHOR_EMAIL set to default
#16 10.84 warning: VERGEN_GIT_COMMIT_AUTHOR_NAME set to default
#16 10.84 warning: VERGEN_GIT_COMMIT_COUNT set to default
#16 10.84 warning: VERGEN_GIT_COMMIT_DATE set to default
#16 10.84 warning: VERGEN_GIT_COMMIT_MESSAGE set to default
#16 10.84 warning: VERGEN_GIT_COMMIT_TIMESTAMP set to default
#16 10.84 warning: VERGEN_GIT_DESCRIBE set to default
#16 10.84 warning: VERGEN_GIT_SHA set to default
#16 10.84    Compiling tabby v0.1.0 (/root/workspace/crates/tabby)
...
```

Reply:
that’s weird - because we do install `git` command in docker image. (gitcl means the library will use `git` cli command to avoid linking a much larger library).

Could you double check if `git` command actually works in docker image?

Reply:
https://github.com/TabbyML/tabby/blob/main/Dockerfile#L39

Looks like the `git` installation is only for prod stage, not for `builder` stage?

Reply:
ah, nice catch.

Then let’s install it in builder stage as well? So `gitcl` should just work.

Reply:
Communicating offline